### PR TITLE
CIV-6238 - Welsh - Bilingual language preference

### PR DESCRIPTION
**Pull request by neeta-hmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-6238

### Change description ###
Added a code changes to save a defendant claim response language preference.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by LeonardoPalmeiro — DISMISSED** [review]

**Review by dtortolaV1 — APPROVED** [review]

File: src/test/unit/services/translation/response/convertToCCDRepondentResponseLanguage.test.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import {CCDRespondentResponseLanguage} from 'models/ccdResponse/ccdRespondentLiPResponse';
+import {ClaimBilingualLanguagePreference} from 'models/claimBilingualLanguagePreference';
+import {Claim} from 'models/claim';
+import {toCCDRespondentResponseLanguage} from 'services/translation/response/convertToCCDRespondentLiPResponse';
+
+describe('translate claimBilingualLanguagePreference to CCD model', () => {
```
Comment (PaulH-HMCTS) [inline src/test/unit/services/translation/response/convertToCCDRepondentResponseLanguage.test.ts:6]: Same comments as above

**Review by LeonardoPalmeiro — DISMISSED** [review]

File: src/test/unit/services/translation/convertToCUI/convertToCUIRespondentLipResponse.test.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      respondent1MediationLiPResponse: undefined,
+      respondent1ResponseLanguage: CCDRespondentResponseLanguage.ENGLISH,
+    };
+    //When
+    const responseLang = toCUIClaimBilingualLangPreference(ccdRespondLipResponse.respondent1ResponseLanguage);
+    //then
+    expect(ClaimBilingualLanguagePreference.ENGLISH).toBe(responseLang);
+
+  });
+});
```
[the remark this answers is not in the document]
Comment (neeta-hmcts) [reply src/test/unit/services/translation/convertToCUI/convertToCUIRespondentLipResponse.test.ts:47]: added test for undefined lang.

File: src/test/unit/services/translation/response/convertToCCDRepondentResponseLanguage.test.ts (hunk 1 above)
Comment (neeta-hmcts) [reply src/test/unit/services/translation/response/convertToCCDRepondentResponseLanguage.test.ts:6]: added test for undefined lang.

File: src/test/unit/services/translation/convertToCUI/convertToCUIRespondentLipResponse.test.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    //Given
+    const ccdRespondLipResponse : CCDRespondentLiPResponse = {
+      respondent1LiPFinancialDetails: undefined,
+      respondent1MediationLiPResponse: undefined,
+      respondent1ResponseLanguage: CCDRespondentResponseLanguage.BOTH,
+    };
+    //When
+    const responseLang = toCUIClaimBilingualLangPreference(ccdRespondLipResponse.respondent1ResponseLanguage);
+    //then
+    expect(ClaimBilingualLanguagePreference.WELSH_AND_ENGLISH).toBe(responseLang);
```
[the remark this answers is not in the document]
Comment (neeta-hmcts) [reply src/test/unit/services/translation/convertToCUI/convertToCUIRespondentLipResponse.test.ts]: fixed.

**Review by PaulH-HMCTS — APPROVED** [review]

**Review by laperla11 — DISMISSED** [review]

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by PaulH-HMCTS — DISMISSED** [review]

**Review by laarmada — APPROVED** [review]

**Review by laperla11 — DISMISSED** [review]

**Pull request merged**